### ISSUE TITLE
Update to latest models

### ISSUE THREAD

**Issue by maribox** [pasquini-dario/LLMmap#1]
Hey!
This tool is great, but I think it would be even greater if you could update it to the latest models!

**Comment by pasquini-dario** [comment]
Yes. I'm planning to release a bigger model trained on more and more modern LLMs. I just need to find the resources to collect the training set.

**Comment by GurvanR** [comment]
Hello, 
I'm a PhD student looking for comparing your fingerprinting method with others.
Thank you very much for updating your repo recently, it's very helpful for our evaluating purpose. However, we're having an issue. We want to use LLMmap in a specific setup that is quite vanilla: T=1.0, freq_pen =0.0 no system prompts etc. and with 22 of your LLMs. We use vllm to make the inferences with
```
'dtype: "bfloat16",
'top_k': 50,
'max_tokens': 500,
'temperature':1.0
```

We plotted the global accuracy as a function of the nb_of_queries, using the default_queries set.
The performances we got were terrible compared to what you have in your paper (see figure in attached files).
Would you have any idea of why this happened? The code is also attached.

Also, why the classifier model provided don't take less than 8 queries? By reading the paper, I thought the model $f$ could take different number of traces while trained on a fixed one. Then to simulate it, we "padded" the rest of required queries with empty ones. 

Second surprise, the performances were better with 7 queries than 8.

Thank you very much for your attention and your help. 
Gurvan

[LLMmap_Reproduction_issue.zip](https://github.com/user-attachments/files/21852717/LLMmap_Reproduction_issue.zip)
